<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Базовые проверки перед использованием инструментов</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QCheckBox" name="map">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Карта открыта</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="selection">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>117</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбраны объекты</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="starting">
      <property name="text">
       <string>Проверить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closing">
      <property name="text">
       <string>Закрыть</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>236</width>
     <height>88</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QCheckBox" name="lines">
      <property name="text">
       <string>Линии</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="polygons">
      <property name="text">
       <string>Многоугольники</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="points">
      <property name="text">
       <string>Точки</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="any_object">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Любой объект</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="only_one">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Ровно один объект</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QRadioButton" name="active_layer">
      <property name="text">
       <string>на выбраном слое</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QRadioButton" name="any_layer">
      <property name="text">
       <string>на любом слое</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closing</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
